### PR TITLE
Fix and update translation template.

### PR DESCRIPTION
**Pull request by sunweaver**

### CODE REVIEW

File: data/org.ayatana.indicator.power.gschema.xml.in [hunk 1]
```diff
@@ -1,24 +0,0 @@
-<schemalist>
```
Comment (tari01) [inline data/org.ayatana.indicator.power.gschema.xml.in:1]: You have deleted this file, instead of doing a rename + replace.

**Review by tari01 — CHANGES_REQUESTED** [review]

File: data/CMakeLists.txt [hunk 2]
```diff
@@ -4,12 +4,6 @@
 
 set (SCHEMA_NAME "org.ayatana.indicator.power.gschema.xml")
 set (SCHEMA_FILE "${CMAKE_CURRENT_BINARY_DIR}/${SCHEMA_NAME}")
```
Comment (tari01) [inline data/CMakeLists.txt:6]: We can also drop SCHEMA_FILE.

**Review by tari01 — CHANGES_REQUESTED** [review]

File: data/CMakeLists.txt (hunk 2 above)
Comment (sunweaver) [reply data/CMakeLists.txt]: Amended.

File: data/org.ayatana.indicator.power.gschema.xml.in (hunk 1 above)
Comment (sunweaver) [reply data/org.ayatana.indicator.power.gschema.xml.in:1]: Ah, forgot to add this to Git. Amended now.

**Pull request merged**